kokoro/ubuntu/new_release.sh

```diff
@@ -23,24 +24,19 @@ test -n "${ANALYTICS_TRACKING_ID}"
 
 cd git/google-cloud-eclipse
 
-# Drop the `.qualifier` in `metadata.product` and use the exact release
-# version. (For example, `version="1.3.0.qualifier"` to `version="1.3.0"`.)
-# version.
-#
-# https://github.com/GoogleCloudPlatform/google-cloud-eclipse/pull/2363#issuecomment-327844378
-# https://github.com/GoogleCloudPlatform/google-cloud-eclipse/issues/2211
-readonly CT4E_DISPLAY_VERSION=$( \
-  xmlstarlet sel -t -v '/product/@version' gcp-repo/metadata.product )
-readonly NO_QUALIFIER_VERSION="${CT4E_DISPLAY_VERSION%.qualifier}"
-xmlstarlet ed --inplace -u '/product/@version' -v "${NO_QUALIFIER_VERSION}" \
-  gcp-repo/metadata.product
+VERSION_SUFFIX_PROPERTY=
+if [ "${PRODUCT_VERSION_SUFFIX}" ]; then
+  VERSION_SUFFIX_PROPERTY="'${PRODUCT_VERSION_SUFFIX}'"
+fi
 
 # Need to unset `TMPDIR` for `xvfb-run` due to a bug:
 # https://bugs.launchpad.net/ubuntu/+source/xorg-server/+bug/972324
 TMPDIR= xvfb-run \
-  mvn -V -B -Doauth.client.id="${OAUTH_CLIENT_ID}" \
-            -Doauth.client.secret="${OAUTH_CLIENT_SECRET}" \
-            -Dga.tracking.id="${ANALYTICS_TRACKING_ID}" \
+  mvn -V -B \
+      -Dproduct.version.qualifier.suffix="${VERSION_SUFFIX_PROPERTY}" \
```

Comment:
I think you could reduce this change to a one-liner with: `${PRODUCT_VERSION_SUFFIX:+-Dproduct.version.qualifier.suffix="'${PRODUCT_VERSION_SUFFIX}'"`.

Reply:
Ah, that's nice.